nips/11.md

```diff
@@ -0,0 +1,61 @@
+NIP-11
+======
+
+Relay Information Document
+---------------------------
+
+`draft` `optional` `author:scsibug`
+
+Relays may provide server metadata to clients to inform them of capabilities, administrative contacts, and various server attributes.  This is made available as a JSON document over HTTP, on the same URI as the relay's websocket.
+
+When a relay receives an HTTP(s) request with an `Accept` header of `application/nostr+json` to a URI supporting WebSocket upgrades, they SHOULD return a document with the following structure.
+
+```json
+["NOSTR_SERVER_INFO",
+{
+  name: <string identifying relay>,
+  descr: <string with detailed information>,
+  pubkey: <administrative contact pubkey>,
+  email: <administrative contact email address>,
+  supported_nips: <a list of NIPS supported by the relay>,
+  software: <string identifying relay software URL>,
+  version: <string version identifier>
+}]
+```
+
+Additional fields may be present, but they MUST be prefixed with `x-` to indicate they are not standard.
```

Comment:
I'm strongly against the `x-` stuff. If people want to return extra fields they should be able to. And if others want to rely on these fields then they are a standard -- even if they're not covered on these documents.

Also software implementations must know reasonably well what they're supposed to return and what they must look for.

Reply:
No problem, I can remove the `x-` suggestion.